.travis.yml

```diff
@@ -5,6 +5,7 @@ branches:
   only:
   - master
 rvm:
+  - 2.6
   - 2.5
   - 2.4
   - 2.3
```

Comment:
We could check in servicesdb if we can kill testing versions we don't rely on internally anymore.

Reply:
I tried checking for it but it's not very easy to do, so I'm going to punt on that :)